sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/models/DigitalTwinsGetByIdResponse.java

```diff
@@ -23,14 +24,14 @@ public DigitalTwinsGetByIdResponse(
             HttpRequest request,
             int statusCode,
             HttpHeaders rawHeaders,
-            Object value,
+            Map<String, Object> value,
             DigitalTwinsGetByIdHeaders headers) {
         super(request, statusCode, rawHeaders, value, headers);
     }
 
     /** @return the deserialized response body. */
     @Override
-    public Object getValue() {
+    public Map<String, Object> getValue() {
```

Comment:
I think this is a side-effect of what the API review committee wanted from us. We'll need to make sure we update the translations where necessary.

Reply:
It was updated automatically through generate script.